### PR TITLE
Migrate scheduled experiments

### PR DESCRIPTION
**Pull request by kk-min**
This PR migrates the scheduled experiments from the old runner to new runners. **This should only be merged after `continuous-benchmarking` has been successfully fast-forwarded (See #304)**, as the new runners and scheduled experiments include other providers (GCR, Azure, etc.).

## Changes
- Update `continuous-benchmarking.yml`

The updated workflow file with engage the new self-hosted runners labeled `aws`, `gcr`, `cloudflare` and `azure` for benchmarking their respective providers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,10 +86,76 @@ Finally, we look at the procedural steps adopted by the framework:
...
+| Go Runtime                                   | Yes        | No                                       | No               | Yes     | Yes                                      | ??**                                     |
+| Java Runtime                                 | Yes        | No                                       | No               | ??**    | No                                       | ??**                                     |
+| Node JS                                      | Yes        | Yes                                      | No               | Yes     | Yes                                      | Yes                                      |
+| HTTP trigger                                 | Yes        | Yes                                      | No               | Yes     | Yes                                      | Yes                                      |
+| Resource Management (e.g. buckets databases) | Yes        | Yes (at least it looks like it)          | No               | ??**    | ??**                                     | ??**                                     |
```
Comment (dilinade) [inline README.md:152]: Do we really need this to be displayed in Readme? 
If so we'll have to make the writing a bit formal

File: README.md [hunk 2]
```diff
@@ -81,10 +86,76 @@ Finally, we look at the procedural steps adopted by the framework:
...
+3. A connection with the serverless vendor is established. This is abstracted away behind a common interface having only four functions: ListAPIs, DeployFunction, RemoveFunction, and UpdateFunction. Used exclusively throughout the codebase, this interface offers seamless integration functionality with any provider.
+4. In the provisioning phase, serverless.com framework is used to deploy the functions to the cloud and to establish HTTP endpoints. The functions are configured based on the experiment JSON file.
 5. The last step runs all the experiments either sequentially or in parallel: bursts are successively sent to each available endpoint, followed by a sleep duration specified by the IAT. The process is repeated until all responses have been recorded to disk. Finally, statistics and visualizations are generated.
 
+## Serverless.com Framework Deployment
+
+Stellar uses serverless.com framework to deploy serverless functions to cloud.
+
+![deployment](design/deployment.jpg)
```
Comment (dilinade) [inline README.md:97]: We should push the 'deployment.jpg' image as well

**Review by dilinade — COMMENTED** [review]
Workflow file LGTM. Added a few comments in readme

File: README.md (hunk 1 above)
Comment (kk-min) [reply README.md]: I've removed the table for now.

**Review by dilinade — APPROVED** [review]

**Pull request merged**